src/main/app/src/app/zaken/zaak-details-wijzigen/zaak-details-wijzigen.component.ts

```diff
@@ -129,7 +133,9 @@ export class CaseDetailsEditComponent implements OnInit, OnDestroy {
       ),
       behandelaar: null,
       communicatiekanaal: this.zaak.communicatiekanaal ?? null,
-      einddatumGepland: moment(this.zaak.einddatumGepland),
+      einddatumGepland: this.zaak.einddatumGepland
```

Comment:
will `zaak.einddatumGepland` be truthy in cases where `zaaktype.servicenorm` is falsy? And should that be taken into account setting the form's `einddatumGepland` value?